scripts/packages/packager/local-entrypoint.sh

```diff
@@ -4,6 +4,13 @@ set -e
 set -x
 set -euxo pipefail
 
+case "$(uname -m)" in
```

Comment:
we already have OSARCH variable in the root Makefile you can leverage. It should take from that value instead of adding more logic here

Reply:
Uhm I don't see any relation between Makefile arg(s), which are not even being exported, and entrypoint scripts. Moreover, with multiarch containers (e.g. buildx) you'll get broken entrypoint for archs not matching host's arch if you will be relying on OSARCH obtained on host. Finally, OSARCH is being weirdly transformed from `uname -m` (introduced here - https://github.com/nginx/agent/commit/24a158b319da1a9acd5bf52eb119d93a327810a5#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L36), so I wouldn't rely on that at all for the sake of simplicity.